sixtyfps_compiler/lexer.rs

```diff
@@ -123,11 +124,21 @@ pub fn lex_string(text: &str, state: &mut LexState) -> usize {
                 return stop + 1;
             }
             b'\\' => {
+                if text_len <= stop + 1 {
+                    return text_len;
```

Comment:
for consistency with the code just a few lines before

```suggestion
                    // FIXME: report an error for unterminated string
                    return 0;
```